Editor/Scripts/create_wind_forces_tutorial.py

```diff
@@ -0,0 +1,69 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+import sys
+import azlmbr
+
+from PySide2.QtWidgets import QMenuBar
+
+from tutorial import Tutorial, TutorialStep
+
+class WindForcesTutorial(Tutorial):
+    def __init__(self):
+        super(WindForcesTutorial, self).__init__()
+
+        self.title = "Create Wind Forces"
+
+        self.add_step(TutorialStep("Create Wind Forces", """<html><p style="font-size:13px">Greetings!<br><br>We can use a <b>PhysX Force Region</b> component to
+                create both global and local wind forces. Wind forces affect entities with components that are affected by wind,
+                such as cloth. Wind forces don't affect <b>PhysX Rigid Body</b> components.
+                For this tutorial, make sure that you have the <b>Nvidia Cloth Gem</b> enabled in your project.
+                <br><br>Click next to continue.</p></html>"""))
```

Comment:
Might want to see what @micronAMZN thinks but as of now since all the tutorials start with `Click next to continue` at the bottom, we could assume the first step is the introduction and then do:
1.  When the `self.current_step_index` is 0 (first step), the counter label is `Click next to continue.`
2.  All other steps display the `Step X of N`

Reply:
Resolving this and adding it as feedback in interactivetutorials_dialog.py.